e2e/alphabet-card-detail.test.ts

```diff
@@ -47,5 +47,23 @@ describe('Detail', () => {
         await expect(element(by.id('w1.mp3'))).toBeVisible();
     });
 
-    //TODO create a test for error handling if audio is not available
+    it('should display an error message if there is a word audio error', async () => {
+        const expectAudioError = true;
```

Comment:
I'm expecting to see some setup code where you attempt to render a bogus URL. How are you controlling whether  there is audio to load?

Reply:
This has been refactored.